renku/cli/.gitlab-ci.yml

```diff
@@ -1,4 +1,10 @@
-# renku: v1.dev20180619
+# renku: v1.dev20180723
+
+variables:
+  GIT_STRATEGY: clone
+  GIT_SSL_NO_VERIFY: "true"
```

Comment:
not sure the quotes are needed?

Reply:
yes, because we want string "true" not YAML boolean `true`.